assets/js/util/i18n.js

```diff
@@ -340,3 +340,16 @@ export const getLocale = ( _global = global ) => {
 
 	return _global.navigator.language;
 };
+
+/**
+ * Gets the current locale language.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} _global The global window object.
+ * @return {string} Current locale language.
+ */
+export function getLocaleLanguage( _global = global ) {
+	const locale = getLocale( _global );
+	return locale.substr( 0, 2 );
+}
```

Comment:
These should definitely only be two-characters? For languages that share the first two characters but are quite different (Brazilian vs Continental Portuguese, Traditional vs Simplified Chinese, etc.) this will result in quite fuzzy language matching for these URLs, won't it?

I know there are redirects for `en-US` to `en`, but if that the case for other locales?

----

I actually just tested this with the Chinese example, and this removes precision:

`https://support.google.com/?hl=zh-TW` redirects to `https://support.google.com/?hl=zh-Hant`
`https://support.google.com/?hl=zh` redirects to `https://support.google.com/?hl=zh-Hans`

So there _are_ differences there; we shouldn't be using only the first two characters for locales.

Reply:
Using `getLocale` should be fine here.

Reply:
From AC:

> The word "Privacy" should link to ...google.com/privacypolicy?hl={**currentTwoLetterLanguageBasedOnLocale**}
> The word "Terms" should link to ...google.com/terms?hl={**currentTwoLetterLanguageBasedOnLocale**}

Reply:
@felixarntz: Can you clarify why that is required/in the ACs, because it seems like it will send users to the wrong URL/locale.

Reply:
@eugene-manuilov  @tofumatt Hmm that's a fair point. Apologies, I had been under the assumption that Google's different locale support only considers the language part - so maybe it's best to revert that change and use both the first segments of the locale string present. It's important though to limit it to the first two segments, not just take the whole string, which at least in PHP side can have more elements, which may cause problems in non-WordPress systems (see #2105). I wonder whether we should generally stick to getting the locale from the server eventually, rather than a mix of server and client, which may be different - we should avoid having mixed language content in Site Kit where possible.

Maybe for this issue/PR:
* Let's check what the client-side locale values can look like. If we're certain they don't include a third segment (like what is possible in WP), we can just take the full string.
* Otherwise, we should strip the client-side locale down to the first two segments only and use that.

I'll update the ACs to reflect that.

Reply:
Thanks, @felixarntz. PR is updated.